pyproject.toml

```diff
@@ -140,3 +140,16 @@ exclude = "(dist/)|(build/)|(.*\\.ipynb)"
 
 [tool.bandit.assert_used]
 skips = ["*/test_*.py"] # assert statements are good practice with pytest
+
+[tool.curlylint]
+include = '\.jinja$'
+
+[tool.curlylint.rules]
+aria_role = true
+django_forms_rendering = true
+html_has_lang = true
+image_alt = true
+# indent = 4
```

Comment:
Had to comment this one out because we are not consistent. A follow-up PR could uncomment this and fix all files, and we could add it to `.gitignore`.